scikeras/utils/transformers.py

```diff
@@ -255,13 +254,12 @@ def inverse_transform(
                 class_predictions[:, idx] = 1
             class_predictions = self._final_encoder.inverse_transform(class_predictions)
         elif self._target_type == "multiclass-onehot":
-            # array([.8, .1, .1], [.1, .8, .1]) ->
-            # array([[1, 0, 0], [0, 1, 0]])
+            # array([.8, .1, .1], [.1, .8, .1]) -> array([[1, 0, 0], [0, 1, 0]])
             idx = np.argmax(y, axis=-1)
             class_predictions = np.zeros(y.shape, dtype=int)
-            class_predictions[:, idx] = 1
+            class_predictions[np.arange(idx.size), idx] = 1
```

Comment:
Style nit: why is this change necessary?

Reply:
This is the actual bug being fixed. Apparently `class_predictions[:, idx]` assigns 1 to every element in the array.

Reply:
Got it. That's because `x[:, [0, 1]] = 1` assigns the columns 0 and 1 to be 1 for every row. `x[idx0, idx1]` will assign `x[idx0[i], idx1[i]] = 1` for every `i` if `idx0` and `idx1` are the same length.

Style nit: maybe use `class_predictions[np.arange(len(idx)), idx] = 1` instead?

Reply:
Yeah that should work fine, I'll implement that change.

Reply:
Changed as suggested.